java/java-guestbook/.gitignore

```diff
@@ -0,0 +1,7 @@
+**/.DS_Store
+.idea
```

Comment:
should we include .idea? How will we add changes that trigger extension installation and so on?

Reply:
yeah, you're right. fixed